### PR TITLE
Update to node 16 and test node 14 in the CI

### PR DESCRIPTION
**Pull request by juancancela**
Updated Node to latest versions using following [script](https://github.com/Financial-Times/next/wiki/Node-14-&-16-Migration-Guide)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,11 +45,11 @@
     }
   },
   "engines": {
-    "node": "12.x",
-    "npm": "7.x"
+    "node": "14.x || 16.x",
+    "npm": "7.x || 8.x"
```
Comment (fenglish) [inline package.json:49]: I'm really not sure we need to add `8.x`. If you use volta when run `npm install`, it should be ok with only `npm 7`

Comment (fenglish) [inline package.json:49]: I'm really not sure we need to add `8.x`. If you use volta when run `npm install`, it should be ok with only `npm 7`

Comment (fenglish) [reply package.json:49]: When I run `npm install` in my local on this repo after node upgrade, no error is thrown.
Screenshot 2022-04-06 at 14 26 45

Comment (juancancela) [reply package.json]: got it! updated.

**Review by fenglish — APPROVED** [review]

**Review by gyss — APPROVED** [review]

**Pull request merged**